mlem/contrib/venv.py

```diff
@@ -0,0 +1,187 @@
+import os
+import platform
+import subprocess
+import sys
+import venv
+from abc import abstractmethod
+from types import SimpleNamespace
+from typing import ClassVar, List, Optional
+
+from mlem.core.errors import MlemError
+from mlem.core.objects import MlemBuilder, MlemModel
+from mlem.ui import EMOJI_OK, EMOJI_PACK, echo
+
+
+def get_python_exe_in_virtual_env(env_dir: str, env_type: str = "pip"):
+    if platform.system() == "Windows":
+        if env_type == "pip":
```

Comment:
This probably should be a boolean flag and not a string

Reply:
done, using `use_conda_env` as True or False now